cmd/run.go

```diff
@@ -98,14 +100,34 @@ func runRun(cmd *cobra.Command, args []string, clientFn runClientFn) (err error)
 		return fmt.Errorf("the given path '%v' does not contain an initialized function", config.Path)
 	}
 
+	// Client for use running (and potentially building)
 	client := clientFn(config)
 
+	// Build if not built and --build
 	if config.Build && !function.Built() {
 		if err = client.Build(cmd.Context(), config.Path); err != nil {
 			return
 		}
 	}
-	return client.Run(cmd.Context(), config.Path)
+
+	// Run the Function at path
+	port, stop, runtimeErrCh, err := client.Run(cmd.Context(), config.Path)
+	if err != nil {
+		return
+	}
+	defer stop()
+
+	fmt.Fprintf(cmd.OutOrStderr(), "Function started on port %v\n", port)
```

Comment:
Thanks!   I am firmly of the belief that a running process should be "as quiet as pssible, but no quieter".  Echoing out the port on which the Function was started _is necessary information!_  My only qualm is that the wording of the output is not conducive to internationalization nor scropting.  I would personally prefer something like:
```
OK http://localhost:123445
```
Because that is a mere ` | awk '{print $2}'` away from being used in a larger command or script (the prefix `OK` a fairly portable ACK...) But I felt that might be going a bit too far?  After all, we seem to have by consensus chosen a generally more verbose, beginner-friendly route thus far.

Reply:
Hmm - I can see your point, but I do think we're trying to optimize for beginner-friendly usability in most cases. Maybe down the road a `--quiet` or `--bare` flag could achieve a similar result?